packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -28,15 +28,18 @@ export const auth = (): Auth => {
       clerkRequest.clerkUrl.searchParams.get(constants.QueryParameters.DevBrowser) ||
       clerkRequest.cookies.get(constants.Cookies.DevBrowser);
 
+    const requestData = getHeader(request, constants.Headers.ClerkRequestData);
+    const decryptedRequestData = requestData ? decryptClerkRequestData(requestData) : {};
```

Comment:
🙃 (nitpicky comment)
It looks like `requestData` is only being used in the ternary operator here. Could we maybe move the conditional logic in `decryptClerkRequestData`, tweak the helper so it handles undefined/null values gracefully (by returning `{}`) and do this instead?

```
const decryptedRequestData = decryptClerkRequestData(getHeader(request, constants.Headers.ClerkRequestData))
```